app/Http/Requests/Api/Application/Allocations/StoreAllocationRequest.php

```diff
@@ -40,7 +40,7 @@ public function validated()
         return [
             'allocation_ip' => $data['ip'],
             'allocation_ports' => $data['ports'],
-            'allocation_alias' => $data['alias'],
+            'allocation_alias' => $data['alias'] !== null ? $data['alias'] : null,
```

Comment:
What is the point of this logic, its literally just the same as using just `$data['alias']`

Reply:
Honest to god, I really don't know/can't remember. It was like 3 months ago. I must've been just expanding the logic for testing and just ended up replacing the test with null and didn't shorten it. I'll commit a change for that. I don't recall there being any specific issue since I made the changes elsewhere.

Maybe it could've been because somewhere the logic was messing up? I'll see if I can pull up my project and double check.